build.xml

```diff
@@ -257,8 +257,10 @@ lib/plexi submodule or add the the command line argument
     description="Run instrumented tests and generate coverage report"/>
 
   <target name="test" depends="build" description="Run JUnit tests">
-    <junit printsummary="yes" haltonfailure="no" failureproperty="test.failure"
-           forkmode="once" fork="true" dir="${basedir}" maxmemory="512m">
+    <property name="build.haltonfailure" value="off"/>
```

Comment:
I renamed it on purpose. The test.* properties are sent through to the tests, and these properties are purely for the build.